src/tasks/dailyItems.ts

```diff
@@ -302,5 +302,15 @@ export const DailyItemTasks: Task[] = [
       completed: () => get("_garbageItemChanged"),
       do: () => cliExecute("fold broken champagne bottle"),
     },
+    {
+      name: "Learn About Bugs",
+      // eslint-disable-next-line libram/verify-constants
+      ready: () => have($item`S.I.T. Course Completion Certificate`),
+      // eslint-disable-next-line libram/verify-constants
+      completed: () => get("_sitCourseCompleted", true) || have($skill`Insectologist`),
```

Comment:
my only small note on this: if prefs get misaligned, and the user doesn't have insectologist but mafia thinks the sit course isn't completed, will this construction lead to an infinite loop?

Reply:
a quick search in the mafia git repository tells me that `indelible-for-a-day` appears here:
https://github.com/kolmafia/kolmafia/blob/881e876fc2f41706259097fa9e41ab34b4779fcc/src/net/sourceforge/kolmafia/request/UseItemRequest.java#L6008-L6012

TL;DR: we're fine